.github/workflows/ci-audio-macos-tcc.yml

```
@@ -0,0 +1,250 @@
+# no cancellation so an in-flight run always completes cleanly.
+concurrency:
+  group: macos-tcc-audio-device
+  cancel-in-progress: false
+
+jobs:
+  macos-tcc-audio:
+    name: "macOS Audio (self-hosted, TCC-granted)"
+    # Fork guard (defense-in-depth; see the security header): the weekly cron
+    # must not fire against a self-hosted runner from a fork context.
+    if: github.repository == 'Codeseys-Labs/rust-crossplat-audio-capture'
+    runs-on: [self-hosted, macos, tcc-audio]
```

actionlint (1.7.12)

[error] label "tcc-audio" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)